src/System.ValueTuple/tests/ValueTupleTests.cs

```diff
@@ -1225,6 +1225,107 @@ public static void EightTuplesWithBadRest()
 #endif
         }
 
+#if netcoreapp11
+        [Fact]
+#endif
+        public static void ValueTupleSerialization()
+        {
+            VerifySerialization(new ValueTuple(),
```

Comment:
These tests are verifying XML serialization.  But the change in question was for runtime serialization, e.g. BinaryFormatter.  Shouldn't the tests be testing that? You can use the Clone helper in https://github.com/dotnet/corefx/blob/master/src/Common/tests/System/Runtime/Serialization/Formatters/BinaryFormatterHelpers.cs to help (and look for other uses of that in other tests as examples).